src/doc/gui_manual/Quantitative/LineSampler.rst

```diff
@@ -0,0 +1,331 @@
+.. _LineSampler:
+
+LineSampler
+-----------
+
+Similar to the Lineout tool one-dimensional curves, created using data
+from 2D or 3D plots, are popular for analyzing data because they are
+simple to compare. However, unlike the Lineout which creates a single
+curve it is a tool, whereas the LineSampler is an operator and can
+create multiple curves.
+
+As an operator the user can define the sampling. That is the user can
```

Comment:
I suggest a rewording:
'Sampling can be defined as a series of "arrays" (e.g. planes) that consist of one or more "channels". For each array ... `